handler/pullRequestHandler.go

```diff
@@ -101,6 +86,57 @@ That's something we need before your Pull Request can be merged. Please see our
 	}
 }
 
+// VerifyPullRequestDescription checks that the PR has anything in the body.
+// If there is no body, an 'invalid' label is added and comment posted to the PR with a link to the contributing guide.
+func VerifyPullRequestDescription(req types.PullRequestOuter, contributingURL string, config config.Config) {
+	ctx := context.Background()
+	token := getAccessToken(config, req.Installation.ID)
+	client := factory.MakeClient(ctx, token, config)
+
+	if req.Action == "opened" {
```

Comment:
is there a `const` defined anywhere for `opened` we could use? I know Richard added a lot of consts earlier and it seemed like a reasonable pattern to follow

Reply:
Good idea! "magic strings" are no fun!

Reply:
const added and updated the places where the string was referenced